package.json

```diff
@@ -47,13 +47,15 @@
     ]
   },
   "devDependencies": {
-    "babel-cli": "^6.16.0",
-    "babel-preset-env": "^1.6.1",
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
What are differences between `@babel/core` and `babel-core`?

Reply:
babel-core `7.0.0-bridge` just reexports `@babel/core`. this is because some packages still depend on babel-core (jest in this case)

see here for more info: https://github.com/babel/babel-bridge